extension/chrome/elements/pgp_block_modules/pgp-block-render-module.ts

```diff
@@ -178,7 +178,7 @@ export class PgpBlockViewRenderModule {
       // eslint-disable-next-line @typescript-eslint/naming-convention
       await AcctStore.set(this.view.acctEmail, { successfully_received_at_leat_one_message: true });
     }
-    const contentWithLink = linkifyHtml(htmlContent);
+    const contentWithLink = Xss.convertHtmlEntities(linkifyHtml(htmlContent));
```

Comment:
@sosnovsky Are you happy with this approach?
I converted `&lt;` to correct `<` so that html can be rendered correctly

Reply:
Hmm, it fixes issue but breaks keeping tags in plain text (as failed test shows).
Did you find any difference in message text when sending simple `reply` versus `reply all`? As `reply` message doesn't show `.cryptup_reply` div

Reply:
or how about just convert `&lt;` to `<` and `&gt;` to `>` using regex or something like that in above function?

Reply:
OK

Reply:
Looks like it happens because FES sends reply messages with `text/plain` body, if I add `text/html` then `div. crypt_up` is hidden, but `cryptup-data` property disappear. Can you please check if we remove such properties when decrypting messages?

Reply:
No, `cryptup-data` property should be there. If message body is `text/plain` then it's present, but if I change it to `text/html` on FES side, then browser extension removes `cryptup-data` property from `div. crypt_up` and it's not possible to parse reply recipients.
So I thought maybe we remove html properties when decrypting messages for security reasons, but not sure

Reply:
Just finished checking. We indeed remove such properties when showing the `pgp_block`.

However, when the user clicks `reply/reply all` and tries to retrieve the `fcReplyToken`, it attempts to get it from the plain decrypted content.
I thin it should work fine when you return `text/html` on the `FES` side.

Reply:
I tested it with `text/html` on FES side, but clicking `reply` can't fetch recipients list as decrypted content doesn't include `cryptup-data`:

<img width="320" alt="Screenshot 2023-06-09 at 12 58 53" src="https://github.com/FlowCrypt/flowcrypt-browser/assets/1062093/e24db4be-cf85-435d-82fb-aa0bd1216634">

Then we should fix this issue on both sides:
- on FES we should use `text/html` instead of `text/plain`
- in browser extension we should add exception for `div.cryptup_reply`, so it's `cryptup-data` property won't be removed

What do you think?

Reply:
Sounds good.
However, I think this one should be considered after FES deploy.

Reply:
Hmm, but then users won't be able to reply to FES messages until they update browser extension to the latest version, as `cryptup-data` will be empty. Need some backwards-compatible solution

Reply:
Maybe we can extract `cryptup-data` before rendering message and then remove `div.cryptup-reply`?
Is it possible or we parse `cryptup-data` when clicking `reply` button?

Reply:
Yep, You are right.
I think we can. Let me implement solution

Reply:
@sosnovsky, @martgil,

I'm not sure how to generate a test email for this case since the current `FES` returns the `text/plain` type.

Could someone please assist me in generating a test email for the above case? The email should have a message body in `text/html` format and include the `cryptup-data` attribute(Message which might be generated by using the "reply all" function to an FES message)

Reply:
I mean we should extract `cryptup-data` from current `text/plain` message. Because if we change message body to `text/html` then users with older extension builds (without this fix) won't be able to reply to these messages, as `cryptup-data` property is removed during decryption for `text/html`.

Reply:
Aha, I see.
That makes sense

Reply:
@sosnovsky, I was confused about the issue of going backward and forward.
However, the current functionality of extracting `cryptup-data` from a `text/plain` message works well.
The problem lies in the visibility of the `cryptup_reply` div due to the presence of `&lt;` and `&gt;` instead of `<` and `>`.

I believe a simple solution would be to replace those two strings with `<` and `>`, respectively. By doing so, the issue should be resolved. What are your thoughts on this?

Reply:
Yeah, it'll be simpler solution than changing logic of message decryption. But it should be just temporary fix, as correct one will be changing FES message body type to `text/html` (as it contains html tags). We can't change type now, as it'll break replying to FES messages in current browser extension version (+ need to check it in mobile apps).

So I propose such steps:
- in this PR implement replacing `&lt;` and `&gt;` with `<` and `>` (maybe just for `div.cryptup_reply`, if it's not too complicated, so it won't affect other messages)
- in this PR update decryption logic of `text/html` to leave `cryptup-data` property in decrypted message (as currently all properties are removed)
- I'll file an issue for FES to change password message replies body type to `text/html`, but it should be implemented after several releases, when all users will have browser extension versions with correct support for `cryptup-data` property

As a result, we'll have a quick fix for this issue, and will prepare browser extension for correct fix in the future.

Reply:
Sounds good!

Reply:
No. @martgil 
Thank you